airbyte_cdk/test/standard_tests/docker_base.py

```diff
@@ -242,6 +243,9 @@ def test_docker_image_build_and_read(
             the local docker image cache using `docker image prune -a` command.
           - If the --connector-image arg is provided, it will be used instead of building the image.
         """
+        if self.is_destination_connector():
```

Comment:
This resolves the `destination-motherduck` failure, where the test fails stating that the connector does not support the "discover" command.